website/blog/2023-09-23-release-0-21.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Announcing Yew 0.21
+authors: [hamza]
+---
+
+The Yew development team is thrilled to unveil Yew 0.21.0, a significant milestone in the journey of empowering developers to create dependable and high-performance web applications with Rust.
+Let's dive into the major highlights of this release.
+
+## What's new
+
+### 1. Changing Signatures: A Shift in Hook Dependencies
+
+One of the significant changes in Yew 0.21 is the adjustment to the signature of hooks that accept dependencies.
+Dependencies used to be passed as the second argument after the closure. However, now they're passed as the first argument before the closure.
+
+```rust
+use_effect_with_deps(deps, move |deps: Vec<i32>| {
+    // Do something with dependencies
+});
+```
+
+The reason behind swapping the order of dependencies in the code snippet is to address a specific use case.
+In situations where the same value is needed both to compute a dependency and to be moved by value into the closure, the new order simplifies the code and improves readability and ergonomics.
+
+This is a big breaking change so we've provided [a way to automate the refactor](https://yew.rs/docs/migration-guides/yew/from-0_20_0-to-0_21_0#automated-refactor)
+
+### 2. Versatile Child Types
+
+Yew now allows you to use any type as children within your components. This means you're no longer limited to just the `Children` type.
+Instead, you can use any type, even `Html` or closures, unlocking patterns such as:
+
+```rust
+html! {
+    <Comp>
+        {|p: RenderProps| html!{<>{"Hello, "}{p.name}</>}}
+    </Comp>
+}
+```
+
+### 3. Agents: A Complete Rewrite
+
+Yew 0.21 brings a complete rewrite of `yew-agent`. This streamlines and simplifies the way workers operate. Here's what you need to know:
+
+-   **Unification of PublicWorker and PrivateWorker:** We've brought PublicWorker and PrivateWorker together, creating a single, consistent worker type that's easier to work with.
```

Comment:
This is only true for standalone agent (using gloo-worker directly). reach (public / private) still exists for workers spawned with a provider.

Reply:
```suggestion
```